app/views/idv/getting_started/show.html.erb

```diff
@@ -0,0 +1,100 @@
+<% title @title %>
+
+<%= render 'shared/maintenance_window_alert' do %>
+  <%= render JavascriptRequiredComponent.new(
+        header: t('idv.getting_started.no_js_header'),
+        intro: t('idv.getting_started.no_js_intro', sp_name: @sp_name),
+      ) do %>
+
+    <% if @current_user&.reproof_for_irs?(service_provider: @current_sp) %>
+    <%= render AlertComponent.new(
+          type: :info,
+          message: t('doc_auth.info.irs_reproofing_explanation'),
+          class: ['margin-bottom-2', 'usa-alert--info-important'],
+        )
+    %>
+    <% end %>
+
+  <%= render AlertComponent.new(
+        type: :error,
+        class: [
+          'js-consent-form-alert',
```

Comment:
I suspect this was carried over from the "Welcome" view, but I don't think this class does anything. It looks like I ought to have removed it as part of 5a86611f58aa38a6162fe2be711035aa707794eb.

Reply:
Great, I'll take that out.

Reply:
Thought about it more, and I'm going to do this as a separate PR, since it's going to change the indentation.

Reply:
From conversation in slack: this component is used if the consent checkbox is not checked and javascript is off. It might not be needed since there is also an error that is shown if javascript is off. This will be resolved in a future PR.